index.html

```diff
@@ -6491,19 +6491,35 @@ <h2>Validation</h2>
         <h3>Credential Type</h3>
 
         <p>
-When a [=verifier=] requests one or more [=verifiable credentials=]
-from a [=holder=], they can specify the type of credential(s) that they would
-like to receive. The type of a credential is expressed via the
-<a href="#types">type</a> property. A [=verifiable credential=] of a specific
-type is expected to contain specific [=properties=] that can be used to
-determine whether or not the [=presentation=] meets a set of processing rules
-that the [=verifier=] is executing. By requesting
-[=verifiable credentials=] of a particular `type`, the
-[=verifier=] is able to gather specific information from the [=holder=],
-which originated  with the [=issuer=] of each [=verifiable credential=],
-that will enable it to determine the next stage of an interaction with a
-[=holder=].
+When a [=verifier=] requests one or more [=verifiable credentials=] from a
+[=holder=], they can specify the <a href="#types">type</a> of credential(s)
+that they would like to receive. Credential types, as well as validation schemas
+for the type and each of its [=claims=], are defined by specification authors
+and are published in places like the [[[VC-SPECS]]].
         </p>
+
+        <p>
+The type of a credential is expressed via the <a href="#types">type</a>
+property. A [=verifiable credential=] of a specific type is expected to contain
+specific [=properties=] that can be used to determine whether or not the
+[=presentation=] meets a set of processing rules that the [=verifier=] is
+executing. By requesting [=verifiable credentials=] of a particular `type`, the
+[=verifier=] is able to gather specific information from the [=holder=], which
+originated  with the [=issuer=] of each [=verifiable credential=], that will
+enable it to determine the next stage of an interaction with a [=holder=].
```

Comment:
Potentially confusing antecedent.
```suggestion
enable the [=verifier=] to determine the next stage of an interaction with a [=holder=].
```